### PR TITLE
ADD: ZDR bias code from RadTraQ

### PR DESCRIPTION
**Pull request by rcjackson**
- [x] Addresses part of #1522 
- [x] Tests added
- [x] Documentation reflects changes

I have adapted the ZDR bias code from RadTraQ into PyART with tests and the given example.

### CODE REVIEW

File: pyart/correct/__init__.py [hunk 1]
```diff
@@ -8,6 +8,7 @@
 from .attenuation import calculate_attenuation  # noqa
 from .attenuation import calculate_attenuation_philinear  # noqa
 from .attenuation import calculate_attenuation_zphi  # noqa
+from .bias_and_noise import correct_bias, correct_noise_rhohv, calc_zdr_offset  # noqa
```
Comment (zssherman) [inline pyart/correct/__init__.py:11]: Looks like correct_noise_rhohv got duplicated in the conflict fix.

File: pyart/correct/bias_and_noise.py [hunk 2]
```diff
@@ -144,6 +145,62 @@ def correct_bias(radar, bias=0.0, field_name=None):
...
+    gatefilter: PyART GateFilter
+        Gatefilter for filtering out data for calculating ZDR bias
+    height_range: tuple
+        The minimum and maximum heights in meters for the scan.
+    zdr_var: string or None
+        The name of the ZDR variable. Set to None to have PyART try to determine this automatically.
+
+    Returns
+    -------
+    obj : dict
```
Comment (zssherman) [inline pyart/correct/bias_and_noise.py:165]: Can we rename obj to something else? Not really necessary, but more so i think obj as a class or pyart or xarray obj and not dict

**Review by zssherman — CHANGES_REQUESTED** [review]
Just some small comments, besides that, looks good!

**Review by zssherman — APPROVED** [review]

**Comment by zssherman** [comment]
Doc failure is unrelated to PR, fixed in mine.

**Comment by zssherman** [comment]
@rcjackson I think we got two correct_bias now, there was one on the line below of the changed one.

**Review by mgrover1 — APPROVED** [review]
This looks great! Thanks for the hard work here @rcjackson @zssherman

**Pull request merged**